hs_core/templates/pages/genericresource.html

```diff
@@ -498,14 +516,20 @@ <h2 id="resource-title">{{ title }}</h2>
                         </div>
                     {% endif %}
                 {% else %}
-                    {% if cm.raccess.public %}
+                    {% if cm.raccess.published %}
+                        <strong class="label-public">&nbspPublished</strong>
```

Comment:
@mauriel133 ```$nbsp;``` is probably more standard just from searching on the web although they do not result in any difference on the page display from my testing, so I will change ```$nbsp``` to ```$nbsp;``` unless I hear your objections.